README.md

```diff
@@ -1,3 +1,10 @@
+|  | Badges | 
+| --- | --- |
+| Module | ![ID](https://img.shields.io/badge/ID-sdl--generator--tools-blue) [![Store](https://img.shields.io/badge/Jahia%20Store-Yes-brightgreen)](https://store.jahia.com/contents/modules-repository/org/jahia/modules/sdl-generator-tools.html) |
+| CI / CD | [![CircleCI](https://circleci.com/gh/Jahia/sdl-generator-tools/tree/master.svg?style=shield&circle-token=b8a70b01d5e9483ebbac3babfa26ae80930a56c7)](https://app.circleci.com/pipelines/github/Jahia/sdl-generator-tools) ![Unit Tests](https://img.shields.io/badge/Unit%20Tests-No-red) ![Integration Tests](https://img.shields.io/badge/Integration%20Tests-No-red) ![Build Snapshot](https://img.shields.io/badge/Build%20Snapshot-Yes-brightgreen) ![Build Release](https://img.shields.io/badge/Build%20Release-No-red) | 
```

Comment:
You can remove the status token for circleci since this repo is public

Reply:
ok